README.md

```diff
@@ -9,7 +9,7 @@ In your project's `.rubocop.yml` file, use `inherit_from:` on the first line to
 
 ```
 inherit_from:
-  - https://wwwin-github.cisco.com/raw/Corona/rubocop/master/rubocop-2.3.yml
+  - https://wwwin-github.cisco.com/raw/Corona/rubocop/master/rubocop-3.3.yml
```

Comment:
I think that this is wrong, because we should not reference anymore rubocop from https://wwwin-github.cisco.com/Corona also there is no such file.

Reply:
I tried with 3.0.yml in the URL and it is not working. Looks like this the correct one https://github.com/CiscoSecurityServices/rubocop/blob/master/rubocop-3.3.yml